3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1019,6 +1019,38 @@ MAIUSCOLO:
 ===== Glossario
 Tutte le occorrenze dei termini contenuti nel glossario sono evidenziati con una G in corsivo a pedice.
 
+===== Variabili Typst per termini ricorrenti
+
+Il gruppo ho predisposto una serie di comandi Typst per riferirsi a termini usati molto frequentemente. In particolare troviamo:
```

Comment:
Non è "una serie di comandi": l'espressione "una serie" è di basso registro e non adatta. Piuttosto riformulare l'intero periodo in "Al fine di uniformare termini ricorrenti, vengono predisposte variabili utilizzabili nella redazione dei documenti. I termini in cosinderazione sono relativi a:"